src/cli/bootstrap/bootstrap.ts

```diff
@@ -94,6 +104,21 @@ async function createAccounts() {
     await createAntelopeAccount({ account: 'demo.tmy' }, signer);
     await createAntelopeAccount({ account: 'eosio.msig' }, signer);
     await createAntelopeAccount({ account: 'eosio.token' }, signer);
+
+    console.log('Create accounts for msig control');
+
+    for (let i = 0; i < govKeys.length; i++) {
+        console.log('Create account', indexToAccountName(i));
+        await createAntelopeAccount({ account: indexToAccountName(i) }, signer);
+    }
+}
+
+// Account names can only create digits 1-5
+function indexToAccountName(index: number): string {
+    if (index > 23) throw new Error('Number too large for current algorithm');
+    const indexBase5 = (index + 1).toString(5);
+
+    return indexBase5 + '.found.tmy';
```

Comment:
why you created function to create account names to numbers?

Reply:
Each of the board members get a account
1.found.tmy
2.found.tmy
3.found.tmy

Antelope names can only contain the digits12345, so that is why there is a function to remove and change any digits 06789